packages/components/inputs/toggle-input/src/toggle-input.styles.ts

```diff
@@ -129,6 +129,8 @@ const Label = styled.label<TStyledLabelProps>`
   position: relative;
   display: inline-block;
   cursor: ${(props) => (props.isDisabled ? 'not-allowed' : 'pointer')};
+  align-self: center;
```

Comment:
@Filpob As discussed, I added the margin to take up the whole focus area. 
<img width="309" alt="Screenshot 2023-03-13 at 13 35 51" src="https://user-images.githubusercontent.com/49066275/224712317-f329a76e-a796-4a86-b781-0c5eef0be99e.png">

My only concern in that the wrapping `<Spacing.Inline>` has a different height now. Should the top bottom overflow also be covered. As an alternative we could do:
 
<img width="294" alt="Screenshot 2023-03-13 at 14 10 27" src="https://user-images.githubusercontent.com/49066275/224712173-886049c5-7e6b-4f33-aedd-c41a05e9d35c.png">

You may notice a slight overflow on top/bottom, but the wrapping `<Spacing.Inline>` doesn’t change the size.